OpenNefia.Content/Resources/Locale/en_US/Elona/Prototypes/Skill.lua

```diff
@@ -0,0 +1,873 @@
+﻿OpenNefia.Prototypes.Skill.Elona = {
+    StatStrength = {
+        Name = "Strength",
+        ShortName = " STR",
+
+        OnDecrease = function(entity)
+            return ("%s%s muscles soften.")
+                :format(name(entity), his_owned(entity))
```

Comment:
Although these aren't used yet, prefix these functions with the locale function namespace (`_`).

```lua
return ("%s%s muscles soften.")
                :format(_.name(entity), _.his_owned(entity))
```

Reply:
changed in 76f6762